src/Compilers/CSharp/Test/Semantic/Semantics/BindingTests.cs

```diff
@@ -3528,55 +3528,55 @@ public void Test()
-                // (15,44): error CS1061: 'string' does not contain a definition for 'ExtensionMethod2' and no extension method 'ExtensionMethod2' accepting a first argument of type 'string' could be found (are you missing a using directive or an assembly reference?)
+                // (15,44): error CS1061: 'string' does not contain a definition for 'ExtensionMethod2' and no accessible extension method 'ExtensionMethod2' accepting a first argument of type 'string' could be found (are you missing a using directive or an assembly reference?)
                 //         var omittedArg2 = "string literal".ExtensionMethod2<>();
                 Diagnostic(ErrorCode.ERR_NoSuchMemberOrExtension, "ExtensionMethod2<>").WithArguments("string", "ExtensionMethod2").WithLocation(15, 44),
                 // (17,31): error CS0305: Using the generic method group 'ExtensionMethod0' requires 1 type arguments
                 //         var omittedArgFunc0 = "string literal".ExtensionMethod0<>;
                 Diagnostic(ErrorCode.ERR_BadArity, @"""string literal"".ExtensionMethod0<>").WithArguments("ExtensionMethod0", "method group", "1").WithLocation(17, 31),
-                // (17,48): error CS1061: 'string' does not contain a definition for 'ExtensionMethod0' and no extension method 'ExtensionMethod0' accepting a first argument of type 'string' could be found (are you missing a using directive or an assembly reference?)
+                // (17,48): error CS1061: 'string' does not contain a definition for 'ExtensionMethod0' and no accessible extension method 'ExtensionMethod0' accepting a first argument of type 'string' could be found (are you missing a using directive or an assembly reference?)
                 //         var omittedArgFunc0 = "string literal".ExtensionMethod0<>;
                 Diagnostic(ErrorCode.ERR_NoSuchMemberOrExtension, "ExtensionMethod0<>").WithArguments("string", "ExtensionMethod0").WithLocation(17, 48),
-                // (18,31): error CS0305: Using the generic method group 'ExtensionMethod1' requires 1 type arguments
-                //         var omittedArgFunc1 = "string literal".ExtensionMethod1<>;
-                Diagnostic(ErrorCode.ERR_BadArity, @"""string literal"".ExtensionMethod1<>").WithArguments("ExtensionMethod1", "method group", "1").WithLocation(18, 31),
                 // (18,13): error CS0815: Cannot assign method group to an implicitly-typed variable
                 //         var omittedArgFunc1 = "string literal".ExtensionMethod1<>;
                 Diagnostic(ErrorCode.ERR_ImplicitlyTypedVariableAssignedBadValue, @"omittedArgFunc1 = ""string literal"".ExtensionMethod1<>").WithArguments("method group").WithLocation(18, 13),
+                // (18,31): error CS0305: Using the generic method group 'ExtensionMethod1' requires 1 type arguments
```

Comment:
>// (18,31): error CS0305: Using the generic method group 'ExtensionMethod1' requires 1 type arguments [](start = 16, length = 101)

It looks like there are no meaningful changes in this file. Consider reverting. Otherwise, please move this error to its original place so that reviewers could easily see that there are no changes in behavior.